tenacity/async.py

```diff
@@ -58,6 +60,6 @@ def call(self, fn, *args, **kwargs):
             elif isinstance(do, DoSleep):
                 result = NO_RESULT
                 exc_info = None
-                yield from self.sleep(do)
+                yield from self.sleep(do, loop=self.loop)
```

Comment:
The problem is that if anyone was using a different sleep function that does not accept a loop argument, that will break them. As @sileht said, you can pass a lambda or a `functools.partial`